openy_prgf/modules/openy_prgf_faq/openy_prgf_faq.libraries.yml

```diff
@@ -2,3 +2,6 @@ paragraph:
   version: 1.x
   js:
     js/paragraph.js: {}
+  dependencies:
+    - core/jquery
+    - core/once
```

Comment:
Isn't that  drupal/once ?
See https://www.drupal.org/project/once

Reply:
@podarok that was before it was in core, I believe. See [JS API overview](https://www.drupal.org/docs/drupal-apis/javascript-api/javascript-api-overview#:~:text=To%20add%20once%20as%20an%20explicit%20dependency%20of%20a%20custom%20library%2C%20add%C2%A0core/onceas%20shown%20below%20in%20MODULE.libraries.yml%20or%20THEME.libraries.yml%3A)

Reply:
no, all good